package.json

```diff
@@ -1276,4 +1276,4 @@
     "vscode-languageclient": "^9.0.1",
     "xml2js": "^0.6.2"
   }
-}
```

Comment:
nit: each PR keeps changing this. Think there is an NPMRC setting to set consistent line ending

Reply:
Will create a separate pull request to enforce this behaviour.